Chess/include/Piece.h

```diff
@@ -0,0 +1,18 @@
+#pragma once
+#include <string>
+
+using std::string;
+class Board;
+class Piece
+{
+	char _initial;
+	bool _color; // 0 for white 1 for black
+
+public:
+	Piece(char initial, bool color);
```

Comment:
the code is not complete i can see only rook and king

Reply:
added the files